backend/dribeho-backend/pom.xml

```diff
@@ -179,5 +222,125 @@
+                    <!--                        <version>${version.jooq}</version>-->
+                    <!--                        <executions>-->
+                    <!--                            <execution>-->
+                    <!--                                <phase>generate-sources</phase>-->
+                    <!--                                <goals>-->
+                    <!--                                    <goal>generate</goal>-->
+                    <!--                                </goals>-->
+                    <!--                            </execution>-->
+                    <!--                        </executions>-->
+                    <!--                        <configuration>-->
+                    <!--                            <jdbc>-->
+                    <!--                                <url>${flyway.url}</url>-->
+                    <!--                                <user>${flyway.user}</user>-->
+                    <!--                                <password>${flyway.password}</password>-->
+                    <!--                                <schema>public</schema>-->
+                    <!--                            </jdbc>-->
+                    <!--                            <generator>-->
+                    <!--                                <database>-->
+                    <!--                                    <name>org.jooq.meta.mariadb.MariaDBDatabase</name>-->
+                    <!--                                    <includes>.*</includes>-->
+                    <!--                                    <inputSchema>public</inputSchema>-->
+                    <!--                                    <outputSchema>public</outputSchema>-->
+                    <!--                                </database>-->
+                    <!--                                <target>-->
+                    <!--                                    <packageName>com.dribeho.database.jooq</packageName>-->
+                    <!--                                    <directory>${jooq.outputdir}</directory>-->
+                    <!--                                </target>-->
+                    <!--                            </generator>-->
+                    <!--                        </configuration>-->
+                    <!--                        <dependencies>-->
+                    <!--                            <dependency>-->
+                    <!--                                <groupId>org.mariadb.jdbc</groupId>-->
+                    <!--                                <artifactId>mariadb-java-client</artifactId>-->
+                    <!--                                <version>3.1.3</version>-->
+                    <!--                            </dependency>-->
+                    <!--                        </dependencies>-->
+                    <!--                    </plugin>-->
```

Comment:
What's going on here...?

Reply:
Confirmed I hadn't gotten this working and needed to debug some issues generating the jooq code.